File: ArrayProblems/Find Majority In Array/FindMajorityInArray.cs
using System;

static int FindMajority(int[] arr,int n)
{
    int maxCount =0;
    int index = -1;

    for (int i = 0; i < n; i++)
    {
        int count =0 ;
        for (int j = 0; j < n; j++)
        {
            if(arr[i] == arr[j])
                count++;

        }
        if (count > maxCount)
        {
            maxCount = count;
            index = i ;

        }
    }

    if( maxCount > n/2)
    {
        return arr[index];
    }
    else
    {
        return -1;
    }
}

static public void Main (){

        int []arr = {1, 1, 2, 1, 3, 5, 1};
        int n = arr.Length;

        Console.WriteLine(findMajority(arr, n));
        Console.ReadLine();
    }
}

File: ArrayProblems/Find Majority In Array/FindMajorityInArray.py

def findMajority(arr,n):
    maxCount = 0
    index =  -1
    for i in range(n):
        count =0
        for j in range(n):
            if (arr[j] == arr[i]):
                count += 1

        if(count > maxCount):
            maxCount = count
            index = i

    if(maxCount > n//2):
        return arr[index]
    else :
        return -1

if __name__ == "__main__":
    arr = [1, 1, 2, 1, 3, 5, 1]
    n = len(arr)
    print(findMajority(arr, n))